app/src/components/disclosure/styles.tsx

```diff
@@ -13,10 +13,18 @@ export const disclosureGroupCSS = css`
 
   // add border between items, only when child is expanded
   > *:not(:last-child) {
-    &[data-expanded] {
+    &[data-expanded="true"] {
       border-bottom: 1px solid var(--ac-global-border-color-default);
     }
   }
+
+  &[data-as-sibling="true"] {
+    > * {
+      &[data-expanded="true"] {
+        border-bottom: 1px solid var(--ac-global-border-color-default);
+      }
+    }
```

Comment:
this is a massively powerful selector. It might be slightly better to have it be class based? Is the * necessary?

Reply:
Removed